data/successors/successor ships.txt

```diff
@@ -979,6 +1018,9 @@ ship "Vujlet"
 		"hull repair rate" 0.875
 		"hull energy" 1.27
 		"hull heat" 1.27
+		"cloak" .05
+		"cloaking fuel" .09
+		"cloaking energy" .75
```

Comment:
```suggestion
```
This makes sense given the description, but my plans for cloaking are that it's something the Successors might rediscover rather than something they start out with. I'll make a note to change the Vujlet's description to make its lack of cloaking ability less ambiguous.